### PR TITLE
blog(release): initial draft Mixxx 2.4 beta release

### PR DESCRIPTION
**Pull request by Swiftb0y**
Just a quick thing I came up so we can discuss details. See comments in the file.

### CODE REVIEW

**Comment by daschuer** [comment]
Thank you. This is a good start. 
I am not sure about the list of features. My original Idea was to highlight only one or a few. Even if we just add some buzz words it is enough to draw first attraction and than reference the changelog (that also needs and overhaul).

**Comment by Swiftb0y** [comment]
I used the [2.3 beta announcement](https://mixxx.org/news/2020-06-07-mixxx-2-3-beta-released/) as a reference, which essentially posts the entire changelog. I think it makes most sense to include a list of all the topics we want to address in the follow up feature blog posts in the beta announcement. That would get people interested in the release and the individual features. I'd like to think that our users are smarter than just falling for a couple buzzwords so I'd prefer to keep the posts clear and honest. If we just wanted a lengthy post full of meaningless buzzwords, we could just AI generate the post and waste the time of all our readers, so I prefer it short and concise. 

I think we should rather focus the discussion on what features we want to highlight exactly here and in the follow up posts.

**Comment by daschuer** [comment]
I have no strong opinion about the list. My original idea was to publish this one quickly without to much discussion about the list. Are you already fine with the current list in the proposal? If yes I can have a second look and than we can advance. 

Should I ask for help creating the follow up blog in the Forum and Zulip or is one else interested to take that action?

**Comment by Swiftb0y** [comment]
> Are you already fine with the current list in the proposal? If yes I can have a second look and than we can advance.

Let me double check. I compiled the list in a hurry by looking at the changelog. It's likely I missed changes in areas I'm not involved in or made them sound more important than they actually are. 

> Should I ask for help creating the follow up blog in the Forum and Zulip or is one else interested to take that action?

Sure go ahead, lets finish discussing the proposed text for that which I posted on zulip.

**Comment by m0dB** [comment]
Could you maybe split
* GUI performance fixes for macOS (use "all-shaders" waveform types).
into 2 points:
* GUI  performances fixes for macOS
* New optimized "all-shaders" waveform types 

It's not a big deal, but I would say the all-shaders waveforms are best for all platforms, not macOS exclusive, and if we can get confirmation that they work for everyone, in the future we could do a lot of cleanup and code reduction.

**Comment by Swiftb0y** [comment]
Mhmm, that kinda implies two blog posts. How about we reward the point to shift the focus:
> * New "all-shaders" Scrolling waveforms resulting in much better GUI performance on all platforms (_especially MacOS_).

File: content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* Support for saving loops as hotcues ("Saved Loops").
+* Save effect chains and then load them into effect units or the QuickEffect
+* Native Apple Silicon support (better performance on M1/M2/etc. devices).
+<!-- how can we summarize sync leader in a single sentence? -->
+* Explicit Sync leader.
+* New controller mapping scripting engine (ES2016 support).
+* GUI performance fixes for macOS (use "all-shaders" waveform types).
+* A bunch of polish for the Library.
+
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md:26]: Do we want to add cover art download?

Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md:26]: I found it too small of a change (at least in terms of end user facing features, not in terms of technical effort included) to get its own entry. I figured we could mention that in the feature blog post that addresses the library changes. 

No objections from me though if you insist it to be included.

**Comment by daschuer** [comment]
Ok than ping me when you think it is ready.

File: content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+title: Mixxx 2.3.5 released
```
Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md]: yikes, almost messed up the title :eyes:

**Comment by Swiftb0y** [comment]
@daschuer done

**Review by daschuer — APPROVED** [review]
Looks good. Thank you.

**Comment by JoergAtGithub** [comment]
The bug tracker link doesn't work. It points to:
`https://mixxx.org/news/2023-05-31-mixxx-2-4-beta-released/github.com/mixxxdj/mixxx/issues`

**Comment by Swiftb0y** [comment]
Thank you. pushed a fix directly. Sorry.

**Comment by JoergAtGithub** [comment]
On the website it's correct now, but the mirror in the forum is still the old one. Can we somehow retrigger the process to update it?

**Pull request merged**